packages/@lwc/engine-dom/scripts/rollup.config.js

```diff
@@ -31,6 +30,14 @@ module.exports = {
     }),
 
     plugins: [
+        {
+            resolveId(importee) {
+                if (importee === '@lwc/renderer-abstract') {
+                    return path.join(__dirname, '../src/renderer.ts');
+                }
+                return null;
+            },
+        },
```

Comment:
I finally figured out how to replace packages in Rollup; I should apply this technique to `perf-benchmarks` instead of using string replacement...